core/app/modules/import-export/assets/js/ui/footer/footer.js

```diff
@@ -0,0 +1,33 @@
+import Grid from 'elementor-app/ui/grid/grid';
+
+import './footer.scss';
+
+export default function Footer( props ) {
```

Comment:
Isn't it a general component?

Reply:
Yes, moved to the App level (the component name was changed to WizardFooter).